libr/core/panels.c

```diff
@@ -3923,7 +3923,10 @@ static char *__get_word_from_canvas_for_menu(RCore *core, RPanels *panels, int x
 		tmp++;
 		i++;
 	}
-	char *ret = r_str_newlen (pos += strlen (padding), i - strlen (padding));
+	char *ret = NULL;
+	if (i - strlen (padding)) {
```

Comment:
use temporary variable to hold this i - strlen and use it in the check below.. but imho this is the same as R_STR_NDUP and this thing seems to be happening multiple times